package.json

```diff
@@ -24,7 +24,7 @@
         "prepare": "husky install"
     },
     "devDependencies": {
-        "@tsconfig/node12": "1.0.11",
+        "@tsconfig/node14": "^1.0.3",
```

Comment:
question (non-blocking): Do we need the caret(^) in this case here? All other (dev) dependencies do not have one.